.travis.yml

```diff
@@ -53,379 +53,17 @@ cache:
 matrix:
   include:
     - os: osx
-      # We request the oldest image we can (corresponding to OSX 10.10) for maximum compatibility.
+      # We request the oldest image we can (corresponding to OSX 10.11) for maximum compatibility.
       # See: https://docs.travis-ci.com/user/reference/osx/#OS-X-Version
-      osx_image: xcode6.4
+      osx_image: xcode8
       language: generic
       env:
         - SHARD="OSX Native Engine Binary Builder"
         - PREPARE_DEPLOY=1
+        - RUST_BACKTRACE=1
       script:
         - ./pants --version && ./build-support/bin/release.sh -n
 
-    - os: linux
```

Comment:
Why are some other shards are removed?

Reply:
We're iterating on getting this shard working, because it's currently failing.